scm/driver/github/issue.go

```diff
@@ -91,13 +91,14 @@ func (s *issueService) Unlock(ctx context.Context, repo string, number int) (*sc
 }
 
 type issue struct {
-	ID      int    `json:"id"`
-	HTMLURL string `json:"html_url"`
-	Number  int    `json:"number"`
-	State   string `json:"state"`
-	Title   string `json:"title"`
-	Body    string `json:"body"`
-	User    struct {
+	ID          int    `json:"id"`
+	HTMLURL     string `json:"html_url"`
+	Number      int    `json:"number"`
+	State       string `json:"state"`
+	Title       string `json:"title"`
+	Body        string `json:"body"`
+	PullRequest pr     `json:"pull_request"`
```

Comment:
The attribute will container diff and link only? If you use the `pr` type it will contain a whole empty struct just filled by the two values.